examples/03_single_template/dotfiles/hello.template

```diff
@@ -0,0 +1,3 @@
+{{!-- vim: set ft=handlebars: --}}
+{{!-- Will resolve to a greeting to the current user --}}
+Hello {{@if {{$PUNKTF_TARGET_FAMILY}} == "windows"}}{{$USERNAME}}{{@elif {{$PUNKTF_TARGET_FAMILY}} == "unix"}}{{$USER}}{{@else}}Unknown{{@fi}}
```

Comment:
replace unix with linux?

Reply:
Nope, [target family](https://doc.rust-lang.org/reference/conditional-compilation.html#target_family) is `unix` and `windows`. [Target os](https://doc.rust-lang.org/reference/conditional-compilation.html#target_os) is `linux`, `windows`, ...
I will try to add documentation for that in the mdbook soon.